### PR TITLE
change how we determine in you're logged in

### PR DESCRIPTION
**Pull request by IngageMaddoxT**
linting changes

### CODE REVIEW

File: crossroads.net/app/go_volunteer/goVolunteer.routes.js [hunk 1]
```diff
@@ -51,8 +51,8 @@
           CmsInfo: CmsInfo,
           Meta: Meta,
           $state: '$state',
-          LoggedIn: function($state, Session) {
-            if (Session.exists('userId')) {
+          LoggedIn: function($state, Session, $cookies) {
```
Comment (silbermm) [inline crossroads.net/app/go_volunteer/goVolunteer.routes.js]: you do not need to inject `$cookies` here, you never use it.

File: crossroads.net/app/go_volunteer/goVolunteer.routes.js [hunk 2]
```diff
@@ -377,23 +378,30 @@
...
       } else {
         GroupConnectors.ByOrgId.query(
           {orgId: GoVolunteerService.organization.organizationId, initiativeId: 1}, function(data) {
           GoVolunteerService.groupConnectors = data;
           deferred.resolve();
-        }, function(err) {
-          console.log(err);
-          deferred.reject();
-        });
+        },
```
Comment (silbermm) [inline crossroads.net/app/go_volunteer/goVolunteer.routes.js]: thanks for fixing my linting errors!

File: crossroads.net/app/go_volunteer/goVolunteer.routes.js [hunk 3]
```diff
@@ -377,23 +378,30 @@
         GroupConnectors.OpenOrgs.query({initiativeId: 1}, function(data) {
           GoVolunteerService.groupConnectors = data;
           deferred.resolve();
-        }, function(err) {
-          console.log(err);
-          deferred.reject();
-        });
+        },
```
Comment (silbermm) [inline crossroads.net/app/go_volunteer/goVolunteer.routes.js]: thanks for fixing my linting errors

File: crossroads.net/app/go_volunteer/goVolunteer.routes.js (hunk 1 above)
Comment (IngageMaddoxT) [inline crossroads.net/app/go_volunteer/goVolunteer.routes.js]: thanks for catching that.  I was using $cookies, then found a "safer" (maybe) way to handle.  I'll remove this!

**Pull request merged**